### ISSUE TITLE
[Regression] Many tools recently broken (e.g. crop)

### ISSUE THREAD

**Issue by wernight** [photopea/photopea#1359]
OS: Chromebook running Chrome 74.

Pretty much any selection tool is affected. Note that I'm using a mouse, like before. I notice this issue started like a week ago.

Demo: 
![bug](https://user-images.githubusercontent.com/145667/56666585-7bdd0080-66ac-11e9-878d-8f8cf11939e9.gif)

**Comment by photopea** [comment]
Hi, are you using Chrome Beta? It is a known issue in Chrome beta, such that all additional elements are "mirrored" vertically.

**Comment by wernight** [comment]
Yes it's Beta.

**Comment by photopea** [comment]
This issue is reported here regularly like every two weeks. You can report it to Chromium, but I am not sure if they will care about bugs in Beta versions.

**Comment by photopea** [comment]
We can not do anything about it. You can try to convince Google to fix this, or switch to a stable browser.

Strangely, this bug has been in Chrome Beta for years, but never gets to a stable version.

**Comment by photopea** [comment]
https://www.reddit.com/r/chromeos/comments/bh1a4h/is_photopea_broken_on_chrome_os/

**Comment by photopea** [comment]
https://github.com/photopea/photopea/issues/1318

**Comment by photopea** [comment]
Hi, can you just quickly check this for me? Go here https://www.photopea.com/cube.html Do you see the Photopea logo on the cube upside down (in Chrome Beta)?

**Comment by wernight** [comment]
Yes it's upside down. Okay so it's not going to affect people on Stable, which is good.

I was on Beta for years and it's the first time that photopea was broken though. That's odd.

It might be worth detecting and showing a notice in PP if it affects all Chrome Beta, or have somewhere easily findable. I searched a bit in bugs but saw no open bug that seemed to fit.

**Comment by photopea** [comment]
I reported it here: https://bugs.chromium.org/p/chromium/issues/detail?id=956460

In general, there can be an unpredictable number of browsers, which have an unpredictable number of bugs, that are present for an unpredictable period of time. If I had to keep track about all these browsers and their bugs (meaning testing Photopea in all versions of all browsers, as they are released) and make Photopea show a message, when a bug appears or disappears, I am afraid, I would have no time for the development itself :(

But I can't really say it is a bug in a browser. Beta versions were never supposed to work correctly. If you decide to use it, be prepared to report many bugs to browser developers. And if you really think it is a bug in Photopea, at least try it in a normal browser first :P
